h5py/_hl/group.py

```diff
@@ -475,10 +475,11 @@ def copy(self, source, dest, name=None,
        Example:
 
         >>> f = File('myfile.hdf5')
```

Comment:
```suggestion
        >>> f = File('myfile.hdf5', 'w')
```

Like in the example you modified.